src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets

```diff
@@ -107,11 +107,16 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <PropertyGroup>
       <TargetPlatformSupported Condition="'$(TargetPlatformIdentifier)' == 'Windows'">true</TargetPlatformSupported>
+      <UseWorkloadsSpecificError Condition="($(TargetPlatformIdentifier) == 'ios' or $(TargetPlatformIdentifier) == 'tvos' or $(TargetPlatformIdentifier) == 'maccatalyst' or $(TargetPlatformIdentifier) == 'android' or $(TargetPlatformIdentifier.StartsWith('browser'))) and '$(MSBuildEnableWorkloadResolver)' != 'true'">true</UseWorkloadsSpecificError>
```

Comment:
Does that mean `net8.0-browser` is ready to be used? @marcpopMSFT

Reply:
I'll defer to @danroth27 I do see it listed in our documentation: https://learn.microsoft.com/en-us/dotnet/standard/frameworks#net-5-os-specific-tfms

Reply:
@lewing Didn't we disable `net8.0-browser` before shipping .NET 8?

Reply:
@danroth27 Apparently, no.

And as it's now in the official documentation, people are starting to use it. So, I assume it won't be disabled/removed in .NET 9?